cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -248,9 +250,10 @@ public Path getMavenConfFolder(boolean legacy) {
    * @return the maven arguments (MVN_ARGS).
    */
   public String getMavenArgs() {
-    Path mavenConfFolder = getMavenConfFolder(false);
+    Path mavenConfFolder = this.context.getMavenConfigurationFolder();
```

Comment:
good finding that you discovered a code duplication here:
* `IdeContext.getMavenConfigurationFolder()`
* `Mvn.getMavenConfFolder(boolean)`

Shouldn't we actually remove the method from `IdeContext` to completely get rid of the redundancy?
This is all not required for this PR, but then the question is why did you change this line of code here at all?

Reply:
Changed it back to the old one and create a new issue: https://github.com/devonfw/IDEasy/issues/1294